diapfl.F90

```diff
@@ -377,15 +377,12 @@ real fri                 ! function of Rig for KPP shear instability
       zm(k)=zgrid(i,j,k)
 !
 ! --- do rivers here because difs is also used for tracers.
-#if defined (USE_NUOPC_CESMBETA)
       if(cpl_orivers.and.cpl_irivers) then
          riv_input = imp_orivers(i,j,1)+imp_irivers(i,j,1)
       else
          riv_input = rivers(i,j,1)
       endif
-#else
       riv_input = rivers(i,j,1)
```

Comment:
line 385 needs to be removed for it to run as intended.

Reply:
thanks, that is right. I will rerun my test and update